<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gb_device_connection">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>276</width>
      <height>326</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 13px;</string>
    </property>
    <property name="title">
     <string>Device Connection</string>
    </property>
    <widget class="QPushButton" name="btn_serial">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>55</y>
       <width>81</width>
       <height>36</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Serial</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_bluetooth">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>55</y>
       <width>81</width>
       <height>36</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Bluetooth</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Connection Type:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>95</y>
       <width>126</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Available devices:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_devices">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>95</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Baudrate</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_baudrate">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>96</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <item>
      <property name="text">
       <string>1000000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btn_connect">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>230</y>
       <width>96</width>
       <height>36</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QWidget" name="led_connect_status" native="true">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>280</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #C21807;
border: 1px solid black;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_connect_status">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>285</y>
       <width>206</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>No device connected</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_device_desc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>185</y>
       <width>256</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>126</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Device description:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
